src/sentry/web/client_config.py

```diff
@@ -77,9 +77,13 @@ def _get_project_key(project_id):
 
 
 def _get_public_dsn():
+
     if settings.SENTRY_FRONTEND_DSN:
         return settings.SENTRY_FRONTEND_DSN
 
+    if not settings.SENTRY_USE_RELAY:
+        return False
```

Comment:
@JoshFerge might be better given the return value below this line 🤷 

```suggestion
        return ""
```

Reply:
yeah good point.